samples/caf_sensor_manager/sysbuild.cmake

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if("${SB_CONFIG_REMOTE_BOARD}" STREQUAL "")
+  return()
+endif()
+
+message(STATUS "Building remote firmware for ${SB_CONFIG_REMOTE_BOARD}")
+
+# Add remote project
+ExternalZephyrProject_Add(
+  APPLICATION remote
+  SOURCE_DIR ${APP_DIR}/remote
+  BOARD ${SB_CONFIG_REMOTE_BOARD}
+)
+set_property(GLOBAL APPEND PROPERTY PM_DOMAINS CPUNET)
+set_property(GLOBAL APPEND PROPERTY PM_CPUNET_IMAGES remote)
+set_property(GLOBAL PROPERTY DOMAIN_APP_CPUNET remote)
+set(CPUNET_PM_DOMAIN_DYNAMIC_PARTITION remote CACHE INTERNAL "")
```

Comment:
```suggestion
```

not used by 54h20

Reply:
yes but this sample will convert to sysbuild asap - I don't want to fully convert it but see no reason why not add this part now.